client/src/partials/reports/stock_integration/stock_integration.js

```diff
@@ -0,0 +1,150 @@
+angular.module('bhima.controllers')
+.controller('stock_integration', [
+  '$scope',
+  '$timeout',
+  '$translate',
+  'connect',
+  'appstate',
+  'validate',
+  'exchange',
+  function ($scope, $timeout, $translate, connect, appstate, validate, exchange) {
+    var session = $scope.session = {},
+      state = $scope.state,
+      allTypes = $scope.allTypes = '0,1';
```

Comment:
I may have missed something, but it doesn't look like you use this variable at all.  Why is it defined and why is it defined on the `$scope` object?